src/styles/sign-up-page.scss

```diff
@@ -321,4 +390,114 @@
   .slide .loginContent {
     width: 188px;
   }
+}
+
+/*
+  Home-Page
+*/
+
+.homepage {
+  .slide .create-target {
+    margin: 16px auto;
+    width: 257px;
+
+    label {
+      text-align: left;
+      padding-left: 3px;
+    }
+
+    input.area {
+      font-family: $body-font-family;
+      font-size: 13px;
+      font-weight: 600;
+      font-style: normal;
+      font-stretch: normal;
+      line-height: normal;
+      letter-spacing: 1.9px;
+      color: $black;
+    }
+
+    input.choose-title {
+      font-family: $body-font-family;
+      font-size: 11px;
+      font-weight: 600;
+      font-style: normal;
+      font-stretch: normal;
+      line-height: normal;
+      letter-spacing: 0.7px;
+      text-align: left;
+      color: $black;
+      padding: 11px 12px;
+    }
+    
+    input.choose-title::placeholder {
+      font-family: $body-font-family;
+      font-size: 11px;
+      font-weight: 600;
+      font-style: normal;
+      font-stretch: normal;
+      line-height: normal;
+      letter-spacing: 0.7px;
+      text-align: left;
+      color: $black;
+      padding: 11px 0;
+    }
+
+    select.customSelect {
```

Comment:
You are using different formats to name classes, the convention is to use all lowercase and dash to separate the words in css, like custom-select or icon-smile, not customSelect like a js variable

Reply:
It´s done!!